uint64 uid                             # Unique identifier
geometry_msgs/Point first_point        # First point of the segment [m]
geometry_msgs/Point last_point         # Last point of the segment [m]
geometry_msgs/Vector3 first_velocity   # Linear velocity [m/s]
geometry_msgs/Vector3 last_velocity    # Linear velocity [m/s]
